### PR TITLE
Trials Project for Code Review

### PR DESCRIPTION
**Pull request by Alfred-KInyua**
Use semantic HTML tags.
 Use CSS selectors correctly.
 Use CSS box model.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css" type="text/css" />
+    <title>Real food made Real Good!</title>
+</head>
+<!-- Header Section -->
+<header> Real food made Real Good!</header>
```
Comment (Alfred-KInyua) [inline index.html:12]: please use the header h1 element. This ensures good semantic flow for your code and is suitable for screen readers.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<!-- Main section -->
+<main>
+    <div><img src="recipes/images/bon.png" alt="images"></div>
+    <h1> Real food made Real Good!</h1>
+    <!--Bottom Navigation section -->
+        <ul>
+            <li><a href="recipes/spinach_salad.html"> Spinach and Strawberry Salad </a></li>
+            <li><a href="recipes/berry.html"> Gordon's Berry Breakfast Drink</a></li>
+            <li><a href="recipes/cream_cheese.html"> Cream Cheese Frosting II </a></li>
+        </ul>
```
Comment (Alfred-KInyua) [inline index.html:22]: Please add a nav tag element in your header section. This makes up for good semantic flow and is suitable for screen readers

File: style.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  grid-gap: 2px;
+  background-color: azure;
+}
+
+header {
+  display: block;
+  padding: 5px;
+}
+
+.wrap {
```
Comment (Alfred-KInyua) [inline style.css:13]: Please remove this section of code, as you do not have any .wrap class.

**Review by Alfred-KInyua — COMMENTED** [review]
#Changes required :recycle: 

please check my comments below and make all required changes 

Submit for a re-review when you are done 

:smile:

**Review by ishimwezachee — APPROVED** [review]
Hi @Alfred-KInyua , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alfred-KInyua** [comment]
@ishimwezachee 
Nice gif. I kind of look forward to these gifs :smile: :smile:

**Pull request merged**